### PR TITLE
Add mapping

### PR DESCRIPTION
**Pull request by emephield**

### CODE REVIEW

File: update_mapping.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+import json
+import re
+import boto3
+from lib import env
+from datetime import date, datetime
+
+#This script create a json
```
Comment (koxon) [inline update_mapping.py]: What does this do?

Comment (koxon) [reply update_mapping.py]: @emephield 24 days later. What those this do ?
We talked about it in scrum. no updates...

**Comment by koxon** [comment]
@emephield what are we waiting for ?

**Comment by koxon** [comment]
This needs to go asap @emephield

**Comment by emephield** [comment]
Updated the readme @koxon

**Comment by koxon** [comment]
@emephield where?

File: README.md [hunk 2]
```diff
@@ -70,6 +70,35 @@ make deploy/DynamoToES DESC="Process DynamoDB stream to ES"
...
+
+
+```
+./update_mapping.py
+```
+
+This script get all dynamostream linked to the function DynamoToES. If your function
+is named differently, change it into the script.
+
+It create a file `lib/table_mappping.json`.
```
Comment (koxon) [inline README.md:99]: it creates

**Pull request merged**